carsus/io/cmfgen/base.py

```diff
@@ -738,7 +738,7 @@ def _get_levels_lines(self, data):
             symbol = convert_atomic_number2symbol(ion[0])
             logger.info(f'Loading atomic data for {symbol} {ion[1]}.')
 
-            lvl = reader['levels']
+            lvl = reader['levels'].copy()
```

Comment:
I had to do this again because the operations done later on change the label names- due to which the label names-IDs mapping is not done properly. This ensures that I am working with the original dataframe sent from the parser instead of the edited one.